charts/cortx/README.md

```diff
@@ -157,8 +157,29 @@ helm uninstall cortx
 | server.persistence.accessModes | list | `["ReadWriteOnce"]` | Persistent volume access modes |
 | server.persistence.size | string | `"1Gi"` | Persistent volume size |
 | server.replicaCount | int | `1` | Number of Server replicas |
+| server.rgw.customLivenessProbe | object | `{}` | Override the default RGW liveness probe with a custom one. |
+| server.rgw.customReadinessProbe | object | `{}` | Override the default RGW readiness probe with a custom one. |
+| server.rgw.customStartupProbe | object | `{}` | Override the default RGW startup probe with a custom one. |
+| server.rgw.livenessProbe.enabled | bool | `false` | Enable the RGW container liveness probe |
+| server.rgw.livenessProbe.failureThreshold | int | `5` | Number of times to retry the liveness probe after it fails, before the pod is marked Unready. |
+| server.rgw.livenessProbe.initialDelaySeconds | int | `10` | Number of seconds to wait before the liveness probe is initiated |
+| server.rgw.livenessProbe.periodSeconds | int | `5` | How often (in seconds) to perform the liveness probe |
+| server.rgw.livenessProbe.successThreshold | int | `1` | Number of times the liveness probe must succeed, after having failed, to be considered successful (must be 1) |
```

Comment:
Just curious... why "must be 1"?

Reply:
Per the [spec](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#configure-probes), this must be 1 for liveness and startup probes.

Reply:
I considered removing it, but kept it for consistency. Actually, the template implementation should updated to use a macro, which is why it's left in.